crops/core/ops.py

```diff
@@ -50,22 +50,23 @@ def get_sequence_alignment(sequence_1, sequence_2, mode='global', open_gap_score
     return alignment_dict
 
 
-def renumber_pdb_needleman(inseq, instr):
+def renumber_pdb_needleman(inseq, instr): #  , seqback=False):
```

Comment:
Not sure what's the point of adding this?

Reply:
That's a message for my future self. It's ok.

Reply:
The convention is to write a comment on the above line starting with the tag `# TODO`. For example this could be done as follows:
python
```
# TODO add seqback argument
def renumber_pdb_needleman(inseq, instr): 
```